R/latexStrWidth.R

```diff
@@ -391,6 +391,11 @@ function( TeXMetrics ){
 
   # Close the LaTeX file, ready to compile
   close( texIn )
+  
+  if (grepl("1251",Sys.getlocale("LC_CTYPE"))) {
+    texIn <- scan(texFile,what=character(),encoding="CP1251",sep="\n")
+    write.table(texIn,file=texFile,fileEncoding="UTF-8",row.names=FALSE,col.names=FALSE,quote=FALSE)
```

Comment:
If you are sure a character vector is encoded in UTF-8, you can `writeLines(..., useBytes = TRUE)`. It is a little bit hackish to use `write.table()` here.